services/polymart/polymart-latest-version.tester.js

```diff
@@ -0,0 +1,15 @@
+import { withRegex } from '../test-validators.js'
+import { createServiceTester } from '../tester.js'
+export const t = await createServiceTester()
+
+t.create('Polymart Plugin (id 323)')
+  .get('/323.json')
+  .expectBadge({
+    label: 'version',
+    message: withRegex(/^(?!not found$)/),
```

Comment:
This would pass even if the badge was something like 'inaccessible' or some other error. We are going to need to use a more specific validator here. I don't know what the requirements are for a polymart version. Are there any docs? If it is pretty loose, maybe something like `isVPlusDottedVersionNClauses` would be a good choice? https://github.com/badges/shields/blob/2fb98897429c9a5bfa2960a85461a2df66654222/services/test-validators.js#L17-L18

Reply:
The version for it doesn't have any specific form, you can check out the documentation at https://polymart.org/wiki/api.